ESSArch_TA/frontend/static/frontend/scripts/controllers/ProfileCtrl.js

```diff
@@ -8,7 +8,6 @@ angular.module('essarch.controllers').controller('ProfileCtrl', function($q, SA,
         sdError: { type: 'danger', msg: $translate.instant('MISSING_SUBMIT_DESCRIPTION') },
         sipError: { type: 'danger', msg: $translate.instant('MISSING_SIP') },
         pmError: { type: 'danger', msg: $translate.instant('MISSING_PRESERVATION_METADATA') },
-        validationError: { type: 'warning', msg: $translate.instant('MISSING_VALIDATION') },
```

Comment:
Shouldn't `$scope.validationAlert` also be deleted?

Reply:
Yes